JS/Script.js

```diff
@@ -1,73 +1,93 @@
 const bookCollection = document.querySelector('#collection');
 const addBook = document.querySelector('#add-book');
-
```

Comment:
This milestone requires to create a class for book objects and add book functionalities only by adding  methods in class.

`
For example: class Books {
  constructor() {
    this.books = [ ];
  }
// A method to add a book to the list
  addBook(title, author) {
   
}
// A method to remove a from from list
  removeBook(bookID) {
    
  }

  // getter method
  getBooks() {
  }
}
`